docs/Researcher/cli-reference/runai-submit-dist-TF.md

```diff
@@ -273,6 +273,10 @@ runai submit-dist tf --name distributed-job --workers=2 -g 1 \
 > 
 > Mount /root/data to NFS path /public/data on NFS server nfs.example.com for read-write access.
 
+#### --configmap-volume name=<name of configmap>,path=<path to mount> ...'
+
+> Mount a `Configmap` object for use as a data volume.
```

Comment:
Configmap --> ConfigMap
here and everywhere

Reply:
```suggestion
> Mount a `ConfigMap` object for use as a data volume.
```

Reply:
```suggestion
> Mount a `ConfigMap` object for use as a data volume.
```